src/v/cloud_storage_clients/s3_client.cc

```diff
@@ -631,7 +631,7 @@ ss::future<bool> s3_client::self_configure_test(const bucket_name& bucket) {
 
 ss::future<> s3_client::stop() { return _client.stop(); }
 
-void s3_client::shutdown() { _client.shutdown(); }
+void s3_client::shutdown() { _client.shutdown_now(); }
```

Comment:
are we using the old `shutdown` method now? maybe it could be removed from the http client class?

Reply:
We are, or rather, the old shutdown method was just inherited from the base transport

```
    using net::base_transport::shutdown;
    void shutdown_now() noexcept {
        _shutdown_now = true;
        shutdown();
    }
```